packages/common/src/util.ts

```diff
@@ -395,3 +395,15 @@ export function chunkArray(arr: any[], size: number) {
  * @returns
  */
 export const isNil = (value: unknown) => value == null;
+
+/**
+ * Upper case first letter (only) of a string
+ * @param word
+ * @returns Word
+ */
+export const capitalize = (word: string) => {
+  // upper case first letter and return as element in array for backwards compatibility
+  const chars = Array.from(word);
```

Comment:
This is perfectly fine, but fwiw `String` is already iterable, so we could achieve the same w/out the `Array.from`.

```js
const [head, ...tail] = word;
return [head.toUpperCase(), ...tail].join('');
```